src/Polytopes/Polyhedron/properties.jl

```diff
@@ -174,7 +124,15 @@ pm::Vector<pm::Rational>
 """
 function vertices(as::Type{T}, P::Polyhedron) where {T}
     if as == Points
```

Comment:
So, this is completely independent of this PR, but: normally, the point of passing a type as an argument like that is that one can use it for method dispatch. So you'd write:
```suggestion
function vertices(as::Type{Points}, P::Polyhedron)
```
And then there'd be no `throw(ArgumentError("Unsupported `as` argument :" * string(as)))` but rather Julia would issue a regular method dispatch failure error. If one really wants to throw that exception, one can provide a fallback method for that:
```julia
function vertices(as::Type{T}, P::Polyhedron) where {T}
  throw(ArgumentError("Unsupported `as` argument $(as)"))
end
```

Reply:
I am in favor of this, these structures only existed and i copied these.